### DISCUSSION TITLE
在12.8.2版本存在问题，又可能会出现“Windows Defender Antivirus Service”服务还在运行的情况下，我已提出解决方案

### DISCUSSION THREAD

**Discussion by gaoliang714** [ionuttbara/windows-defender-remover#210] [General]
在12.8.2版本存在问题，有可能会出现“Windows Defender Antivirus Service”服务还在运行的情况下，我已提出解决方案，如下
1.使用"Windows"键+ "R"键盘快捷键打开运行命令。
2.键入"regedit.exe"，然后单击确定以打开注册表。
3.浏览以下路径：
"HKEY_LOCAL_MACHINE/SOFTWARE/Policies/Microsoft/Windows Defender "
4.右键单击"Windows Defender"（文件夹）键，选择新建，然后单击DWORD（32位）值。
5.将密钥命名为"DisableAntiSpyware"，然后按Enter键。
6.双击新创建的键并将值设置为0到1。
7.点击OK。

There is an issue in version 12.8.2 where the 'Windows Defender Antivirus Service' service may still be running. I have proposed a solution as follows
1. Use the "Windows" key and the "R" keyboard shortcut to open the run command.
2. Type "regedit. exe" and click OK to open the registry.
3. Browse the following path:
"HKEY_LOCAL_MACHINE/SOFTWARE/Policies/Microsoft/Windows Defender "
4. Right click on the "Windows Defender" (folder) key, select "New", and then click on the 32 bit WORD value.
5. Name the key "DisabAntiSpyware" and press Enter.
6. Double click the newly created key and set the value to 0 to 1.
7. Click OK.

**Comment by cisko99za** [comment]
yes, your trick work fine. hope dev add this into DefenderRemover.exe on next release

**Reply by Web-Cooler** [reply]
我用13版本清理了 你让创建的那个项目 他本来就在注册表 我把1改成0了 不知道有没有效果
